subsys/debug/Kconfig

```diff
@@ -43,6 +45,91 @@ config TRACING
 	  Enable system tracing. This requires a backend such as SEGGER
 	  Systemview to be enabled as well.
 
+if TRACING
+
+config TRACING_SYNC
+	bool "Synchronous Tracing"
+	help
+	  Enable synchronous tracing. This requires very little latency
```

Comment:
"This requires the backend to be very low-latency"

Reply:
@mped-oticon thanks